tests/BUILD

```diff
@@ -0,0 +1,13 @@
+load(":bazel_tests.bzl", "md5_sum")
+
+md5_sum(
+  name = "all_rules",
+  srcs = ["//go:all_rules"],
+  visibility = ["//visibility:private"],
+)
+
+filegroup(
```

Comment:
nope, and for no good reason....
For some reason using the md5 rule directly in a runtime data list does not work, but wrapping it in a filegroup first does.